app/Helpers/database/player-game.php

```diff
@@ -382,22 +381,14 @@ function getGameRecentPlayers(int $gameID, int $maximum_results = 10): array
 {
     $retval = [];
 
-    // determine the most recent session for each user who has played the game
-    $subquery = PlayerSession::where('game_id', $gameID)
-        ->groupBy('user_id')
-        ->select([DB::raw('MAX(rich_presence_updated_at) as rich_presence_updated_at'), 'user_id']);
-
-    // and use that information to get the recent session data for each user
-    $sessions = PlayerSession::where('game_id', $gameID)
-        ->join(DB::raw('(' . $subquery->toSql() . ') as subq'), function ($join) use ($subquery) {
-            $join->on('player_sessions.user_id', '=', 'subq.user_id')
-                 ->on('player_sessions.rich_presence_updated_at', '=', 'subq.rich_presence_updated_at')
-                 ->addBinding($subquery->getBindings());
+    $sessions = GameRecentPlayer::with(['user' => function ($query) {
```

Comment:
Do you need the filter function on both the `with` and `whereHas` clauses?

Reply:
Nope - it's redundant. Fixed in latest. Additionally, I've changed the filter to no longer rely on legacy permissions and instead check for truthiness of `email_verified_at`.

Reply:
In latest, I've changed this to check nullishness of `banned_at`:
```php
$sessions = GameRecentPlayer::with('user')
    ->where('game_id', $gameID)
    ->whereHas('user', function ($query) {
        $query->whereNotNull('banned_at');
    })
    ->orderBy('rich_presence_updated_at', 'DESC');
```

This should hopefully get us the functionality we're looking for without needing to check legacy permissions.